src/Common/src/Interop/Unix/Interop.Libraries.cs

```diff
@@ -6,13 +6,15 @@ internal static partial class Interop
     private static partial class Libraries
     {
         internal const string Libc = "libc";                   // C library
-        internal const string LibCoreClr= "libcoreclr";        // CoreCLR runtime
+        internal const string LibCoreClr = "libcoreclr";        // CoreCLR runtime
         internal const string LibCrypto = "libcrypto";         // OpenSSL crypto library
+        internal const string LibSsl = "libssl";               // OpenSSL library
         internal const string Zlib = "libz";                   // zlib compression library
 
         // Shims
         internal const string SystemNative = "System.Native";
-        internal const string HttpNative   = "System.Net.Http.Native";
+        internal const string HttpNative = "System.Net.Http.Native";
+        internal const string SslNative = "System.Net.Security.Native";
```

Comment:
Do we need a separate native library from System.Security.Cryptography.Native for this?